include/range/v3/view/stride.hpp

```diff
@@ -135,24 +135,35 @@ namespace ranges
             // stride_view const models Range if Rng const models Range, and
             // either (1) Rng is sized, so we can pre-calculate offset_, or (2)
             // Rng is not Bidirectional, so it does not need offset_.
+#ifdef RANGES_WORKAROUND_MSVC_711347
+            static constexpr bool const_iterable = Range<Rng const>() &&
+                (SizedRange<Rng const>() || !BidirectionalRange<Rng const>());
+#else // ^^^ workaround / no workaround vvv
```

Comment:
Really? MSVC can't handle a constexpr static member function?

Reply:
Since we have two parsers, everything in MSVC is context-sensitive. The `constexpr` static member functions explode when used in the constraints for `begin` and `end`, yet the variable templates are fine. I don't even try to understand anymore, I just file the bugs.